src/core/imported/addrlib/CMakeLists.txt

```diff
@@ -212,6 +212,27 @@ if((CMAKE_CXX_COMPILER_ID MATCHES "GNU|[Cc]lang")
 
     # [GCC] Options to Request or Suppress Warnings
     #   https://gcc.gnu.org/onlinedocs/gcc-8.1.0/gcc/Warning-Options.html
+    target_compile_options(addrlib PRIVATE
+        -Wall
+        -Wextra
+        -Wno-unused
+        -Wno-unused-parameter
+        -Wno-ignored-qualifiers
+        -Wno-missing-field-initializers
+        -Wno-implicit-fallthrough)
+
+elseif(CMAKE_CXX_COMPILER_ID STREQUAL "[Cc]lang")
```

Comment:
This also should be 'MATCHES' and will never be true right now, both because 'STREQUAL' doesn't take a regex, and because Clang will always fall into 'GNU|[Cc]lang' above. We really should keep as much common as possible between these anyhow.